components/profile/follow/UserRow.vue

```diff
@@ -68,6 +63,10 @@ const showFollowing = ref(false)
 
 const { urlPrefix } = usePrefix()
 
+const userAddressWithPrefix = computed(() =>
```

Comment:
what about `prefixUserAddress`

Reply:
Updated to `prefixUserAddress`. I don't have any preference for its naming.